_episodes/02-Basics_and_exploring_tuples.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Physics background"
+title: "Basic and exploring our tuples"
```

Comment:
Basics? Or perhaps "Basic data exploration"?

Reply:
Thanks, we will start pushing changes to start to have some progress and be able to see the progress of the other members, this is not going to be the final name, but is more close.